osg_configure/modules/baseconfiguration.py

```diff
@@ -106,6 +106,7 @@ def log(self, mesg, **kwargs):
                 message = "Option '%s' in section '%s' located in %s: " % (kwargs['option'],
                                                                            kwargs['section'],
                                                                            file_location)
+                message += "\n" + " " * 9
         message += mesg
```

Comment:
The indentation you added will only apply to the first line and look weird if it `mesg` is ever mult-line